package.json

```diff
@@ -56,29 +56,29 @@
     "ajv-keywords": "^3.4.0",
     "body-parser": "^1.18.3",
     "chai": "^4.2.0",
-    "chai-sinon": "^2.8.1",
```

Comment:
This is an old dependency that does not support newer versions of sinon, replaced with more up-to-date "sinon-chai"